common/config/src/main/java/io/helidon/common/config/ConfigBuilderSupport.java

```diff
@@ -26,7 +26,10 @@
 /**
  * Methods used from generated code in builders when
  * {@link io.helidon.builder.api.Prototype.Configured} is used.
+ *
+ * @deprecated use {@code io.helidon.config.ConfigBuilderSupport} instead
  */
+@Deprecated(forRemoval = true, since = "4.3.0")
 @SuppressWarnings("OptionalUsedAsFieldOrParameterType")
 public final class ConfigBuilderSupport {
```

Comment:
Should we make `io.helidon.common.config.ConfigBuilderSupport` extend `io.helidon.config.ConfigBuilderSupport` and have the generated configured builders still use `io.helidon.common.config.ConfigBuilderSupport` to preserve "forward compatibility" ?

Would that only leave the new dependency as the only breaking change ?